services/queue/config.yml

```diff
@@ -5,6 +5,8 @@ defaults:
     # Number of seconds before the claim to a run expires, defaulting to 1200
     claimTimeout:                 !env:number:optional TASK_CLAIM_TIMEOUT
 
+    maxTaskDeadline:              !env:number:optional MAX_TASK_DEADLINE
```

Comment:
will it be in seconds or in days? I saw some message below that it might be days. Would be good to leave some comment here as well

Reply:
maybe would be even better just to name it `maxTaskDeadlineDays` or so to avoid confusions

Reply:
👍 I updated the name